src/org/zaproxy/zap/extension/zest/ZapAddOn.xml

```diff
@@ -7,7 +7,7 @@
 	<url>https://github.com/zaproxy/zap-core-help/wiki/HelpAddonsZestZest</url>
 	<changes>
 	<![CDATA[
-	Minor code changes.<br>
+	Show always the expected URL in request statements (Issue 2854).<br>
```

Comment:
Haha no, the edit should be the order of the first two words...I shouldn't have changed the other, must have been distracted.